packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,8 +10,18 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+-   [**] [Embed block] Add the top 5 specific embed blocks to the Block inserter list [#34967]
 -   [**] Embed block: Detect when an embeddable URL is pasted into an empty paragraph. [#35204]
 
+1.62.2
+------
+* Same as 1.62.1 but with the changelog.
+
+1.62.1
+------
+* [**] Image block: fix height and border regression. [https://github.com/WordPress/gutenberg/pull/34957]
+* [**] Column block: fix width attribute flout cutoff. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921]
```

Comment:
These entries have a different format from the rest of the changelog and reference GB-mobile, I'm wondering if there was an issue when updating with `trunk` 🤔 .

Reply:
Fixed the release notes issue since we noticed it here 0334b75